gamedata/scripts/magazines.script

```diff
@@ -142,6 +142,10 @@ function get_sec_chambered(wpn)
 		-- print_dbg("there are %s rounds in wep, set first round to %s", weapon:get_ammo_in_magazine(), first_round)
 end
 
+function retain_round(weapon)
+	return get_config("retain_round") and magazine_binder.is_open_bolt_weapon(weapon)
```

Comment:
Not sure I understand the logic correctly, but open bolt weapons should not retain round in chamber regardless of setting?
```suggestion
	return get_config("retain_round") and not magazine_binder.is_open_bolt_weapon(weapon)
```